### ISSUE TITLE
Does not respect netlify.toml?

### ISSUE THREAD

**Issue by screendriver** [nwtgck/actions-netlify#168]
Could it be that this GitHub action does not respect configurations in [netlify.tom](https://docs.netlify.com/configure-builds/file-based-configuration/)? For example I have defined following redirect rule

```toml
[[redirects]]
  from = "https://me.com/*"
  to = "https://www.me.io/:splat"
  status = 301
  force = true
```

This redirect worked as expected before I switched from using `netlify-cli` in my npm scripts manually to this GitHub action 🤔

**Comment by nwtgck** [comment]
@screendriver Thanks.  I'll try to implement.

**Comment by nwtgck** [comment]
@screendriver I implemented the feature.

You can try  with `uses: nwtgck/actions-netlify@b9d0ce523c192042da6e5f582e5f5f4ee3190756`. Let me know whether that is what you want or not.

**Comment by screendriver** [comment]
It works! 🎉

**Comment by nwtgck** [comment]
Thanks for your confirmation! I'll merge that.

**Comment by nwtgck** [comment]
@screendriver This feature waas released in `v1.1.1` now. You can use this with `@v1`, `@v1.1` or `@v1.1.1`.

**Comment by screendriver** [comment]
Thanks! :+1:
